<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>c2bCiterWidget</class>
 <widget class="QWidget" name="c2bCiterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>c2bCiter - cb2Bib</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="Line" name="line1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="c2bCiterView" name="citationsView">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>7</number>
     </property>
     <item>
      <widget class="QStatusBar" name="statusBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeGripEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="filterIcon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="cb2bib.qrc">:/icons/icons/filter.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="clipboardIcon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="cb2bib.qrc">:/icons/icons/clipboard.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="Line" name="line3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>c2bCiterView</class>
   <extends>QTableView</extends>
   <header>c2bCiterView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cb2bib.qrc"/>
 </resources>
 <connections/>
</ui>
